--- Makefile	2021-08-30 14:24:54.561536600 +0200
+++ Makefile_	2021-08-30 14:27:39.796070400 +0200
@@ -10,10 +10,10 @@
 # CONFIG := msys2-64
 
 # features (the more the better)
-ENABLE_TCL := 1
-ENABLE_ABC := 1
+ENABLE_TCL := 0
+ENABLE_ABC := 0
 ENABLE_GLOB := 1
-ENABLE_PLUGINS := 1
+ENABLE_PLUGINS := 0
 ENABLE_READLINE := 1
 ENABLE_EDITLINE := 0
 ENABLE_GHDL := 0
@@ -658,6 +658,7 @@
 
 include $(YOSYS_SRC)/backends/verilog/Makefile.inc
 include $(YOSYS_SRC)/backends/rtlil/Makefile.inc
+include $(YOSYS_SRC)/backends/pexlif/Makefile.inc
 
 include $(YOSYS_SRC)/techlibs/common/Makefile.inc
 
