--- Makefile    2022-02-03 17:00:45.864633700 +0100
+++ Makefile_VossII     2022-02-03 17:44:37.497000998 +0100
@@ -10,10 +10,10 @@
 # CONFIG := msys2-64
 
 # features (the more the better)
-ENABLE_TCL := 1
+ENABLE_TCL := 0
 ENABLE_ABC := 1
 ENABLE_GLOB := 1
-ENABLE_PLUGINS := 1
+ENABLE_PLUGINS := 0
 ENABLE_READLINE := 1
 ENABLE_EDITLINE := 0
 ENABLE_GHDL := 0
@@ -142,7 +142,7 @@
 # is just a symlink to your actual ABC working directory, as 'make mrproper'
 # will remove the 'abc' directory and you do not want to accidentally
 # delete your work on ABC..
-ABCREV = f6fa2dd
+ABCREV = default
 ABCPULL = 1
 ABCURL ?= https://github.com/YosysHQ/abc
 ABCMKARGS = CC="$(CXX)" CXX="$(CXX)" ABC_USE_LIBSTDCXX=1 VERBOSE=$(Q)
@@ -668,6 +668,7 @@
 
 include $(YOSYS_SRC)/backends/verilog/Makefile.inc
 include $(YOSYS_SRC)/backends/rtlil/Makefile.inc
+include $(YOSYS_SRC)/backends/pexlif/Makefile.inc
 
 include $(YOSYS_SRC)/techlibs/common/Makefile.inc
 
